src/libraries/Common/src/Interop/Windows/Kernel32/Interop.GetComputerNameEx.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Kernel32
+    {
+        [DllImport(Libraries.Kernel32, EntryPoint = "GetComputerNameExW", SetLastError = true, CharSet = CharSet.Unicode, ExactSpelling = true)]
+        // The documentation for GetComputerNameEx says type BOOL but returns zero or nonzero rather than 0 or 1.
+        // To avoid hacked bools and potential bad code generation; we compare the result with 0 explicitly.
```

Comment:
We can delete this comment.  This is true for many Win32 APIs, and we assume it in all of our P/Invokes.
```suggestion
```